experimental/PropertyDDS/packages/property-shared-tree-interop/src/schemaConverter.ts

```diff
@@ -294,11 +292,19 @@ export function convertPSetSchemaToSharedTreeLls(
 		} else {
 			const fieldKind =
 				splitTypeId.context === "array" ? FieldKinds.sequence : FieldKinds.optional;
-
+			const localFieldTypes = new Set<TreeSchemaIdentifier>([brand(splitTypeId.typeid)]);
```

Comment:
This code block addition is correct, I only do not understand how it worked before (it generated correctly the array of inherited types even without this block.